README.md

```diff
@@ -228,9 +233,46 @@ all cases, but for the 2NT bids we use Puppet Stayman.
 * 3♥️!/3♠️! Preemptive raise. Less than invitational with 4+ card support.
 * 4♥️!/4♠️! To play, no slam interest.
 
-## After 1NT or 1♣️!-1♦️!-1NT
+## After 10-12 1NT
 
-* 2♣️ Stayman, invitational values
+* 2♣️ Stayman, generally invitational but garbage stayman allowed. With garbage
+  stayman style hands you can also consider waiting for opponents to double and
+  then utilize the runout system.
+* 2♦️! Game forcing Stayman
+* 2♥️ To play
+* 2♠️ To play
+* 2NT! Weak with both minors
+* 3♣️!/3♦️!/3♥️!/3♠️! Preemptive natural
+* 3NT To play
+* 4♣️! Texas transfer to hearts
+* 4♦️! Texas transfer to spades
+* 4♥️ To play
+* 4♠️ To play
+* 4NT Quantitative invite to slam
+* 5♣️ To play
+* 5♦️ To play
+
+### Runouts after 1NT-X or 1NT-P-P-X-P-P
+
+Over a double, runouts are akin to our strong notrump defense.
+Runouts are on regardless of whether the double is meant as penalty or not.
+
+* Pass Suggests playing 1NTx
```

Comment:
What does "suggests" mean? Leave as 1NTx unless there is a strong preference by either player not to?

Reply:
Yes